### PR TITLE
Fix all phpstan level 6 issue for test dir

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

File: tests/Action/Baz.php [hunk 1]
```diff
@@ -15,6 +15,9 @@
 
 class Baz
 {
+    /**
+     * @var Bar
```
Comment (jordisala1991) [inline tests/Action/Baz.php:19]: shouldnt you allow null?

File: tests/Fixtures/Bundle/Entity/Foo.php [hunk 2]
```diff
@@ -15,25 +15,43 @@
 
 class Foo
 {
+    /**
+     * @var mixed
```
Comment (jordisala1991) [inline tests/Fixtures/Bundle/Entity/Foo.php:19]: mixed includes null?

**Review by jordisala1991 — COMMENTED** [review]
Good work there! I left some minor comments, and also I think we should go always for Class&Mock. I saw some mixing between Class&Mock and Mock&Class

File: tests/Fixtures/Bundle/Entity/Foo.php (hunk 2 above)
Comment (VincentLanglet) [reply tests/Fixtures/Bundle/Entity/Foo.php:19]: Yes

File: tests/Action/Baz.php (hunk 1 above)
Comment (VincentLanglet) [reply tests/Action/Baz.php:19]: Done

**Comment by OskarStark** [comment]
@jordisala1991 this could be a nice addition for CS fixer, WDYT?

**Comment by VincentLanglet** [comment]
I rebased and now tests are green.

We might add some rule about the `&`, I would prefer to do it in another PR if it's ok for you @jordisala1991
Using a linter could be better indeed.

**Review by jordisala1991 — APPROVED** [review]

**Review by phansys — APPROVED** [review]

**Pull request merged**